bullet_train/app/controllers/concerns/account/invitations/controller_base.rb

```diff
@@ -76,7 +76,7 @@ def accept
   # POST /invitations/1/resend
   def resend
     @invitation = Invitation.find_by(uuid: params[:id])
-    if @invitation
+    if @invitation.touch
```

Comment:
I think this line is causing the test failures. I think the `touch` should probably be inside the conditional instead of _being_ the conditional.

```suggestion
    if @invitation
      @invitation.touch
```